protoc-builder/Makefile

```diff
@@ -51,13 +54,27 @@ protos: proto-image
 		--openapiv2_out=${MOUNT_POINT}/${OPENAPI_OUT} \
     ${PROTOS}
 
-.PHONY: proto-image
-proto-image: Dockerfile
+python: Dockerfile.python protoc-python-image
+	@echo "Generating python protos"
+	@mkdir -p ${PROJECT_DIR}/${CLIENTS_OUT}/python
+	${DOCKER_RUN} -v ${PROJECT_DIR}:${MOUNT_POINT} ${PROTOC_PYTHON_IMAGE} \
+		-I/opt/include -I/googleapis -I/protobuf-specs -I${MOUNT_POINT}/api/proto \
+    --python_betterproto_opt=pydantic_dataclass \
+	  --python_betterproto_out=${MOUNT_POINT}/${CLIENTS_OUT}/python \
+    ${PROTOS}
+
+.PHONY: protoc-python-image
+protoc-python-image: Dockerfile.python protoc-image
+	@echo "Generating python proto Docker image"
+	${DOCKER_BUILD} -t ${PROTOC_PYTHON_IMAGE} -f Dockerfile.python \
+    --build-arg PROTOC_BASE=${PROTOC_REKOR_IMAGE} \
+    .
+
+.PHONY: protoc-image
+protoc-image: Dockerfile
 	@echo "Generating go proto Docker image"
 	${DOCKER_BUILD} -t ${PROTOC_REKOR_IMAGE} -f Dockerfile \
-    --build-arg BUILDKIT_SYNTAX=docker/dockerfile:1 \
```

Comment:
What's the reason for dropping this? I'm not that familiar with Buildkit, but I think this improves build performance

Reply:
It's a weird commandline override of the syntax. I think we actually need to set the syntax in a comment on each dockerfile: https://docs.docker.com/build/buildkit/frontend/

to docker/dockerfile:1

Reply:
Good find, thanks